Documentation/project-docs/linux-performance-tracing.md

```diff
@@ -89,7 +102,7 @@ Traces are best viewed using PerfView on Windows.  Note that we're currently loo
 2. Download PerfView from <http://aka.ms/perfview>.
 3. Run PerfView.exe
 
-	> ```bash
+	> ```
```

Comment:
@jasonwilliams200OK, thanks for the feedback.  I removed bash from this block as the command is actually targeted for Windows.  Do you know if there is a Windows cmd descriptor that should be used here?